mssqlcli/jsonrpc/tests/test_jsonrpcclient.py

```diff
@@ -260,6 +280,9 @@ def test_stream_closed_during_process(self):
                 str(exception), u'I/O operation on closed file.')
             # Verify response thread is dead.
             self.assertFalse(test_client.request_thread.is_alive())
```

Comment:
just for my understanding... in first test (line 207-208) we shutdown and then check for thread.. and here we check before shutdown. Why the difference?

Reply:
Not positive. In any case it's a broken test, should sync with Brian about this when he's back.